doc/examples/doc_code/raysgd_torch_signatures.py

```diff
@@ -0,0 +1,121 @@
+# flake8: noqa
+"""
+This file holds code for the Pytorch Trainer creator signatures.
+
+It ignores yapf because yapf doesn't allow comments right after code blocks,
+but we put comments right after code blocks to prevent large white spaces
+in the documentation.
+"""
+# yapf: disable
+
+# __torch_model_start__
+import torch.nn as nn
+
+def model_creator(config):
+    """Constructor function for the model(s) to be optimized.
+
+    You will also need to provide a custom training
+    function to specify the optimization procedure for multiple models.
+
+    Args:
+        config (dict): Configuration dictionary passed into ``PyTorchTrainer``.
+
+    Returns:
+        One or more torch.nn.Module objects.
+    """
+    return nn.Linear(1, 1)
```

Comment:
Is this supposed to completely ignore `config`?

Reply:
I see, `config` is also user-specified and this is just a simple example that hardcodes the model creator. Might help to either set a trivial config option or else leave a comment as to why it's hardcoded (because it's a toy example).

Reply:
done, good idea